--- whitebox_tools-2.2.0-vendor.tar.gz/num-bigint/src/biguint/convert.rs
+++ whitebox_tools-2.2.0-vendor.tar.gz/num-bigint/src/biguint/convert.rs
@@ -67,7 +67,7 @@
         .iter()
         .rev()
         .map(|b| (*b as BigDigit).to_u64().unwrap())
-        .div_ceil(&big_digit::BITS.into())
+        .div_ceil(big_digit::BITS.into())
         .collect::<Vec<_>>();
 
     // Bit shift the buffer from the right to remove any leading zeroes.
